amundsen_application/static/js/components/common/ResourceListItem/SchemaInfo/index.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { OverlayTrigger, Popover } from 'react-bootstrap';
+
+import './styles.scss';
+
+export interface SchemaInfoProps{
+  schema?: string;
```

Comment:
Currently using the `?` means that all of these values are optional. However it's worth considering scoping this to what our current use cases are in order to simplify implementation. Essentially while we like to make things flexible, making it too flexible has the effect of creating extra unnecessary work...like have to make sure you check for `null`.

I think it is fair to make `schema` required given that Amundsen is designed to expect this information elsewhere. If you choose to make it required, then you can also remove it from the `defaultProps` below since a required value won't need a default.